test/support/api/patients.js

```diff
@@ -92,3 +92,22 @@ Cypress.Commands.add('routePatientSearch', (mutator = _.identity) => {
   })
     .as('routePatientSearch');
 });
+
+Cypress.Commands.add('routePatientEngagementStatus', (mutator = _.identity, status) => {
```

Comment:
because the data is so simple I was suggesting not using a mutator..  Then no `_.identity` is needed when used.   If there's some reason to keep the mutator we can drop the 2nd param and just use the mutator.